platform/api.visual/examples-readme.txt

```diff
@@ -20,19 +20,27 @@
 
 In January NetBeans has moved to Mercurial.
 The old project structure has been split into 3 places.
+The project has moved to Apache and Git.
+
+EXAMPLES and INTEGRATION EXAMPLES are on external to Apache repository
```

Comment:
something is missing here.

`... are hosted outside of the Apache repository`?

Reply:
should be fixed :p